packages/elements-core/src/components/Docs/HttpService/HttpService.tsx

```diff
@@ -20,8 +19,8 @@ const enhanceVersionString = (version: string): string => {
 
 export type HttpServiceProps = DocsComponentProps<Partial<IHttpService>>;
 
-const HttpServiceComponent = React.memo<HttpServiceProps>(({ className, data, headless }) => {
-  const { search, pathname } = useLocation();
+const HttpServiceComponent = React.memo<HttpServiceProps>(({ className, data, headless, location = {} }) => {
+  const { search, pathname } = location;
```

Comment:
Can we maaybe pass in what we actually need, and not the entire `Location` object? Would be more versatile that way. But of course you can likely prove that this is what we need here.

Reply:
I somewhat disagree.

If we decide that Docs components can have logic depending on location, I don't see a reason to not provide the whole location object in one go. 

If it turns out that other parameters are not necessary, no harm will be done.

If in the future more stuff will be necessary, it will be already there.

But I don't feel strongly about that either way :shrug: